src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,24 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
```

Comment:
The field 'excavatorBucket' should have an access modifier to encapsulate the field properly. Consider using private modifier and providing public getter and setter methods for access.